snapshot/bazel/filer.go

```diff
@@ -0,0 +1,41 @@
+package bazel
+
+import (
+	"github.com/twitter/scoot/snapshot"
+	"github.com/twitter/scoot/snapshot/snapshots"
+	"os/exec"
+)
+
+func MakeDefaultBzFiler() *bzFiler {
+	return &bzFiler{
+		command: "fs_util",
+		updater: snapshots.MakeNoopUpdater(),
+	}
+}
+
+func MakeBzFilerWithLocalStore(localStorePath string) *bzFiler {
+	return &bzFiler{
+		command:        "fs_util",
+		updater:        snapshots.MakeNoopUpdater(),
+		localStorePath: localStorePath,
+	}
+}
+
+// Satisfies snapshot.Checkouter, snapshot.Ingester, and snapshot.Updater
+type bzFiler struct {
+	command        string
+	updater        snapshot.Updater
+	localStorePath string
+	// Not yet implemented:
+	// bypassLocalStore bool
+	// skipServer       bool
+	// serverAddress    string
```

Comment:
FYI this is implemented in https://github.com/pantsbuild/pants/pull/5179 (but currently only used for read operations). Write operations coming soon!

Reply:
As soon as we support write operations (probably next week), I'll start publishing compiled fs_util binaries somewhere useful :)

Reply:
cool! looking forward to giving it a whirl